tests/ctl/nmstatectl_test.py

```diff
@@ -36,11 +36,23 @@
 }
 """
 
+
 EMPTY_JSON_STATE = """{
     "interfaces": []
 }
 """
 
+EMPTY_YAML_STATE = """---
```

Comment:
We could convert json to yaml or in reverse, not sure if it is a good step.

Reply:
I would like to keep it as is at at least for these parts to express the intended formatting. With conversions we would need to make sure that they match as well.